packages/system/src/components/homepage/homepage.tsx

```diff
@@ -103,9 +103,15 @@ export function Homepage({
 					<Blogs blogs={blogs} />
 				</div>
 				<ContributorBanner contributors={contributorsData}></ContributorBanner>
-				<div className={homepageTwoAndOneSectionStyle}>
+				<div
+					className={
+						books.length !== 0
+							? homepageTwoAndOneSectionStyle
+							: ''
+					}
+				>
 					<LatestTools tools={tools} />
-					<Books books={books} />
+					{books.length !== 0 && <Books books={books} />}
```

Comment:
Could reduce the conditional checks and consolidate like so:

```suggestion
				{books.length > 0 ? (
					<div className="homepageTwoAndOneSectionStyle">
						<LatestTools tools={tools} />
						<Books books={books} />
					</div>
				) : (
					<LatestTools tools={tools} />
				)}
```